semantic_conventions.md

```diff
@@ -0,0 +1,82 @@
+# Semantic Conventions
+
+The [OpenTracing Specification](./specification.md) describes the overarching language-neutral data model and API guidelines for OpenTracing. That data model includes the related concepts of **Span Tags** and **(structured) Log Fields**; though these terms are defined in the specification, there is no guidance there about standard Span tags or logging keys.
+
+Those semantic conventions are described by this document. The document is divided into two sections: first, tables listing all standard Span tags and logging keys; then guidance about how to combine these to model certain important semantic concepts.
+
+### Versioning
+
+Changes to this file affect the OpenTracing specification version. Additions should bump the minor version, and backwards-incompatible changes (or perhaps very large additions) should bump the major version.
+
+## Standard Span tags and log fields
+
+### Span tags table
+
+Span tags apply to **the entire Span**; as such, they apply to the entire timerange of the Span, not a particular moment with a particular timestamp: those sorts of events are best modelled as Span log fields (per the table in the next subsection of this document).
+
+| Span tag name | Type | Notes and examples |
+|:--------------|:-----|:-------------------|
+| `component` | string  | The software package, framework, library, or module that generated the associated Span. E.g., `"grpc"`, `"django"`, `"JDBI"`. |
+| `db.instance` | string | Database instance name. E.g., In java, if the jdbc.url=`"jdbc:mysql://127.0.0.1:3306/customers"`, the instance name is `"customers"`. |
+| `db.statement` | string | A database statement for the given database type. E.g., for `db.type="SQL"`, `"SELECT * FROM wuser_table"`; for `db.type="redis"`, `"SET mykey 'WuValue'"`. |
+| `db.type` | string | Database type. For any SQL database, `"sql"`. For others, the lower-case database category, e.g. `"cassandra"`, `"hbase"`, or `"redis"`. |
+| `db.user` | string | Username for accessing database. E.g., `"readonly_user"` or `"reporting_user"` |
+| `error` | bool | `true` if and only if the associated Span is in an error state |
```

Comment:
would this be more concrete: "if the application considers the operation represented by the Span to have failed"